planning/behavior_path_planner/autoware_behavior_path_static_obstacle_avoidance_module/include/autoware/behavior_path_static_obstacle_avoidance_module/helper.hpp

```diff
@@ -375,8 +381,10 @@ class AvoidanceHelper
     const auto constant_distance = getFrontConstantDistance(object);
     const auto avoidance_distance = getMinAvoidanceDistance(desire_shift_length);
 
-    return object.longitudinal < prepare_distance + constant_distance + avoidance_distance +
-                                   parameters_->wait_and_see_th_closest_distance;
+    const bool enough_distance =
+      object.longitudinal < prepare_distance + constant_distance + avoidance_distance +
+                              parameters_->wait_and_see_th_closest_distance;
+    return std::make_pair(enough_distance, false);
```

Comment:
[memo] if merging or deviating object is closer than threshold, the ego avoids it without operator support.